gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java

```diff
@@ -132,20 +132,7 @@ private String build(final CourseGrade courseGrade) {
 
 		// percentage
 		// not shown in final grade mode
-		final String calculatedGrade;
-		if (this.showOverride && StringUtils.isNotBlank(courseGrade.getEnteredGrade())) {
-
-			// if mapping doesn't exist for this grade override (mapping may have been changed!), map it to 0.
-			// TODO this should probably inform the instructor
-			Double mappedGrade = this.gradebook.getSelectedGradeMapping().getGradeMap().get(courseGrade.getEnteredGrade());
-			if (mappedGrade == null) {
-				mappedGrade = new Double(0);
-			}
-			calculatedGrade = FormatHelper.formatDoubleAsPercentage(mappedGrade);
-
-		} else {
-			calculatedGrade = FormatHelper.formatStringAsPercentage(courseGrade.getCalculatedGrade());
-		}
+		final String calculatedGrade = FormatHelper.formatStringAsPercentage(courseGrade.getCalculatedGrade());
```

Comment:
So, does mappedGrade end up being set at the bottom of the grade boundary, and that was the problem? Does formatStringAsPercentage pick the middle of the grade range?

Reply:
Right, old functionality would find the percentage corresponding to the overridden grade which would confuse instructors.

New behavior just uses the student's calculated grade. So you would see C (59.01%) bolded indicating the instructor was nice and gave you a C but your calculated grade was only 59.01%

Reply:
So, enteredGrade remains the override that the instructor has applied and calculatedGrade is just as it should be, the calculated grade. Sounds good to me.

Reply:
Yes, this is what was decided by T&L as desired behavior. Seems reasonable